README.md

```diff
@@ -7,17 +7,23 @@ use [Bundler] to install the necessary dependencies to run and build the website
 
 Ruby is required to use Bundler so first make sure you have Ruby on your machine.  If you are using
 an OS packaged version of Ruby, you will have to also install the ruby-dev (Ubuntu) or
-ruby-devel (Fedora) package as well.
+ruby-devel (Fedora) package as well. Depending on your OS, you may also need other packages, such as 
+ruby-full, make, gcc, nodejs, build-essentials, or patch.
```

Comment:
Everything after this line can now be reverted back to what it was before.

Reply:
I made the requested change.

Reply:
The diff still shows the other changes.